### PR TITLE
Update instruction for sharpenedblade fedora ISO.

### PR DESCRIPTION
**Pull request by sharpenedblade**
So I forgot to change this with the last PR. Also I wanted to clear up any potential confusion about which ISO people were using.

### CODE REVIEW

File: docs/distributions/fedora/installation.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Many thanks to Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora).
 
-If you need a more updated kernel, use the iso from [here](https://github.com/sharpenedblade/t2linux-fedora-iso/releases). Follow the instalation instructions below until step 5, then install like a normal Fedora iso. If you do not know how to install Fedora normally, read [this](https://docs.fedoraproject.org/en-US/fedora/latest/install-guide/). Remember to follow the [Wi-Fi guide](https://wiki.t2linux.org/guides/wifi-bluetooth/).
```
Comment (AdityaGarg8) [inline docs/distributions/fedora/installation.md:5]: Follow the installation instructions......

Why did you remove this line?

Also, what exactly have you made clear in this PR?

Comment (sharpenedblade) [reply docs/distributions/fedora/installation.md:5]: >Why did you remove this line?

Becuase with the updated instruction you have to follow everything for my ISO.
>Also, what exactly have you made clear in this PR?

Like 2 people dmed me about mikeeqs ISO, and thought it was mine, so I made the link say `ISO from sharpenedblade`.

**Review by AdityaGarg8 — APPROVED** [review]
